### DISCUSSION TITLE
vuepress-plugin-search-pro 引用启动白屏

### DISCUSSION THREAD

**Discussion by zhuonixian** [vuepress-theme-hope/vuepress-theme-hope#4175] [Q&A] [answered]
# 环境信息
@vuepress/client 2.0.0-beta.67
@vuepress/plugin-catalog 2.0.0-rc.28
@vuepress/plugin-git 2.0.0-rc.0
@vuepress/plugin-photo-swipe 2.0.0-rc.28
@vuepress/plugin-search 2.0.0-rc.0
nodejs-jieba 0.1.2
vue 3.4.25
vuepress 2.0.0-beta.67
vuepress-plugin-search-pro 2.0.0-rc.11
vuepress-theme-hope 2.0.0-beta.237
# node -v
v20.12.2
# pnpm -v
9.1.2
CentOS Linux release 8.1.1911 (Core)

# 问题描述
在配置文件引用 vuepress-plugin-search-pro 时，启动项目后，访问服务白屏无反馈显示。

[nohup.txt](https://github.com/vuepress-theme-hope/vuepress-theme-hope/files/15416341/nohup.txt)

[截图.zip](https://github.com/vuepress-theme-hope/vuepress-theme-hope/files/15416346/default.zip)

**Comment by zhuonixian** [comment]
nohup  是项目debug启动的日志

**Comment by zhuonixian** [comment]
config.ts
```ts
  plugins: [ 
    searchProPlugin({
      indexContent: true
    }),
  ],
````

**Reply by Mister-Hope** [reply]
see faq

**Reply by zhuonixian** [reply]
Sorry, it may be a simple question that has delayed your time here. But I swear I've tried every possible way, I have watched the FAQ several times I have also compared the configurations of other projects There are still some issues...

**Reply by Mister-Hope** [reply]
The first thing you should do is you should unify all the plugin and theme  versions, to ensure no warnings about peerDependecies is triggered.

**Reply by Mister-Hope** [reply, marked as answer]
https://theme-hope.vuejs.press/faq/common-error.html#issues-with-peer-dependencies-found

**Reply by zhuonixian** [reply]
Sorry   Habitually ignoring the warning information generated by PNPM has been resolved

**Comment by bihell** [comment]
确实有白屏现象还挑两次。不过部署到正式环境就没这个问题了，所以不太 care

**Comment by zhuonixian** [comment]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/45945591/c6a07cc7-ace4-4963-ac59-1c4f72fb4d9b)
console截图重新再粘下  和正常对比   看着多了几条Warn  Error  
估计还是环境配置的问题、
chunk-DHNSDB55.js?v=48c1519b:114 Uncaught (in promise) Error: useRouteLocale() is called without provider.
